### PR TITLE
appropriately pass multiple command line arguments

### LINKED ISSUE #236: passing multiple command line arguments does not work
e.g. 
```
bazel run n1:main -- action --mongo mongodb://192.168.99.100:27017/
INFO: Analysed target //n1:main (0 packages loaded).
INFO: Found 1 target...
Target //n1:main up-to-date:
  bazel-bin/n1/main-layer.tar
INFO: Elapsed time: 0.213s, Critical Path: 0.00s
INFO: Build completed successfully, 2 total actions

INFO: Running command line: bazel-bin/n1/main action --mongo mongodb://192.168.99.100:27017/
Loaded image ID: sha256:208a32b27e423f3633de625dfa07facc41983a1110709422d5734808552e21a1
Tagging 208a32b27e423f3633de625dfa07facc41983a1110709422d5734808552e21a1 as bazel/n1:main
/home/zaspire/.cache/bazel/_bazel_root/1010357ee0e0c546a5a0c1dee8f8184b/execroot/ava/bazel-out/local-fastbuild/bin/n1: line 233: [: too many arguments
```

### PR DESCRIPTION
**Pull request by Zaspire**
Fixes #236

### CODE REVIEW

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

File: container/layers.bzl [hunk 1]
```diff
@@ -142,7 +142,7 @@ def incremental_load(ctx, images, output,
     ]
     if run:
       run_statements += [
-          "docker run %s %s \"$@\"" % (run_flags, tag_reference)
+          "docker run %s %s $*" % (run_flags, tag_reference)
```
Comment (mattmoor) [inline container/layers.bzl]: I believe you should revert this line per: https://stackoverflow.com/questions/1695819/pass-bash-script-parameters-to-sub-process-unchanged

Comment (Zaspire) [reply container/layers.bzl]: This way all parameters will be passed as single argv.

e.g. script 1.sh
```bash
echo $1
```
```
→ bash 1.sh "1 2 3"
1 2 3
0s tmp
→ bash 1.sh 1 2 3
1
```

**Comment by Zaspire** [comment]
> Thanks, I think we only need half of this. Can you confirm?

yes.

**Comment by Zaspire** [comment]
PR updated

**Comment by mattmoor** [comment]
Test this please

(for Bazel CI)

**Comment by mattmoor** [comment]
Thanks!

**Pull request merged**